src/hotspot/cpu/x86/vtableStubs_x86_64.cpp

```diff
@@ -176,10 +176,12 @@ VtableStub* VtableStubs::create_itable_stub(int itable_index) {
   // (various calling sequences use r[cd]x, r[sd]i, r[89]; stay away from them)
```

Comment:
We forgot to add `r14` to the comment about used registers. But that was already the case in the original change, so I think it's fine too leave it as is.

Reply:
Thank you for the review. Yes, it looks right to add comments upstream.